update subpages of homepage with i18n setup

laliwa opened the pull request:
1、update subpages of homepage with i18n setup.
2、update Community China members list.

autumnfound commented on layouts/shortcodes/home/call-to-actions.html:
```diff
@@ -16,17 +15,17 @@
                 <div class="featured-section-cta-item-img">
                     <img src="/images/home/get-informed.svg" alt="Get Informed">
```
We should update the alts as these can be read by screenreaders. Otherwise there may be English words on Chinese pages for screen readers which would be confusing

laliwa replied:
Got it, it is a start and has a lot to go, including that the main menu for Chinese doesn't align with the English version, etc. We will fix that step by step.

autumnfound replied:
I'll hold off on merging until you give me the green light then?

laliwa replied:
> I'll hold off on merging until you give me the green light then?

If that is OK, please merge these updates first, thanks.

autumnfound reviewed (commented):
Other than the one piece of feedback which is pretty minor this looks great! I'm guessing this is just the start of the translation process as there are still some pages that don't have translation such as https://deploy-preview-1362--jakartaee.netlify.app/zh/about/working-group/ and https://deploy-preview-1362--jakartaee.netlify.app/zh/connect/

There are additional issues such as sidebar incorrectly loading on one of the immediate subpages (https://deploy-preview-1362--jakartaee.netlify.app/zh/membership/), but we should capture that separately as this issue exists in production

ericpoirier commented:
@laliwa would it be possible for you to rebase this PR? It currently contains conflicts.

Thanks.

ivargrimstad committed:
parent 93090cb312d6818bb0af2d37e4a7dc26dfd29623
author Ivar Grimstad <ivar.grimstad@eclipse-foundation.org> 1643446541 +0100
committer Eric <40311795@qq.com> 1643769110 +0800

parent 9641fedcd08ef3fa83ca5b75d4d7b95e52bb03f0
author Ivar Grimstad <ivar.grimstad@eclipse-foundation.org> 1643446541 +0100
committer Eric <40311795@qq.com> 1643731488 +0800

Update dependend namespaces for EE10 (#1389)

update description according to the change of Eng page.

Signed-off-by: laliwa <40311795@qq.com>

create && update faq.json in order to keep update with the en version

Signed-off-by: laliwa <40311795@qq.com>

add some description about how to involve in the Jakarta EE Community China.
add QR code to connect participants who want to join a Wechat Group.

Signed-off-by: laliwa <40311795@qq.com>

fix spelling of description

Signed-off-by: laliwa <40311795@qq.com>

remove files

Signed-off-by: laliwa <40311795@qq.com>

update member list of community China

Signed-off-by: Eric <40311795@qq.com>

update homepage/calltoactions with i18n setup

Signed-off-by: Eric <40311795@qq.com>

update homepage with i18n setuup including featured-story,members,tesimonials,videos,whats-new

Signed-off-by: Eric <40311795@qq.com>

update china community member list

Signed-off-by: Eric <40311795@qq.com>

update pages translation associate directly with the main menu

/pages/resources.html
fix bug : some anchor id does not match the main menu links

/layouts/release
fix  bug : version page without toolbar cannot switch locale

Signed-off-by: Eric <40311795@qq.com>

change link to eclipse.org mailing list sign-up

Signed-off-by: laliwa <40311795@qq.com>

create && update faq.json in order to keep update with the en version

Signed-off-by: laliwa <40311795@qq.com>

add some description about how to involve in the Jakarta EE Community China.
add QR code to connect participants who want to join a Wechat Group.

Signed-off-by: laliwa <40311795@qq.com>

fix spelling of description

Signed-off-by: laliwa <40311795@qq.com>

remove files

Signed-off-by: laliwa <40311795@qq.com>

update homepage/calltoactions with i18n setup

Signed-off-by: Eric <40311795@qq.com>

update homepage with i18n setuup including featured-story,members,tesimonials,videos,whats-new

Signed-off-by: Eric <40311795@qq.com>

update pages translation associate directly with the main menu

/pages/resources.html
fix bug : some anchor id does not match the main menu links

/layouts/release
fix  bug : version page without toolbar cannot switch locale

Signed-off-by: Eric <40311795@qq.com>

update description according to the change of Eng page.

Signed-off-by: laliwa <40311795@qq.com>

create && update faq.json in order to keep update with the en version

Signed-off-by: laliwa <40311795@qq.com>

add some description about how to involve in the Jakarta EE Community China.
add QR code to connect participants who want to join a Wechat Group.

Signed-off-by: laliwa <40311795@qq.com>

fix spelling of description

Signed-off-by: laliwa <40311795@qq.com>

update memberlist

Signed-off-by: laliwa <40311795@qq.com>

remove files

Signed-off-by: laliwa <40311795@qq.com>

update member list of community China

Signed-off-by: Eric <40311795@qq.com>

update homepage/calltoactions with i18n setup

Signed-off-by: Eric <40311795@qq.com>

update homepage with i18n setuup including featured-story,members,tesimonials,videos,whats-new

Signed-off-by: Eric <40311795@qq.com>

update china community member list

Signed-off-by: Eric <40311795@qq.com>

update pages translation associate directly with the main menu

/pages/resources.html
fix bug : some anchor id does not match the main menu links

/layouts/release
fix  bug : version page without toolbar cannot switch locale

Signed-off-by: Eric <40311795@qq.com>

laliwa commented:
> rebase

Sure, but  I see no conflicts,  maybe someone already dealt with it?

> @laliwa would it be possible for you to rebase this PR? It currently contains conflicts.
> 
> Thanks.

Sure, but  I see no conflicts,  maybe someone already dealt with it?

ericpoirier commented:
> > rebase
> 
> Sure, but I see no conflicts, maybe someone already dealt with it?
> 
> > @laliwa would it be possible for you to rebase this PR? It currently contains conflicts.
> > Thanks.
> 
> Sure, but I see no conflicts, maybe someone already dealt with it?

By pulling the jakartaeechina repo and pulling your PR, I'm able to rebase against the origin/src. I can see that there is a few commits with conflicts, I tried to fixed them but there are some commits that contains chinese text changes and I don't know what is good and what is not good.

Let me know if you need help rebasing your PR.

laliwa commented:
> > > rebase
> > 
> > 
> > Sure, but I see no conflicts, maybe someone already dealt with it?
> > > @laliwa would it be possible for you to rebase this PR? It currently contains conflicts.
> > > Thanks.
> > 
> > 
> > Sure, but I see no conflicts, maybe someone already dealt with it?
> 
> By pulling the jakartaeechina repo and pulling your PR, I'm able to rebase against the origin/src. I can see that there is a few commits with conflicts, I tried to fixed them but there are some commits that contains chinese text changes and I don't know what is good and what is not good.
> 
> Let me know if you need help rebasing your PR.

I had tried to do this,  pls check if things going well

ericpoirier commented:
> > > > rebase
> > > 
> > > 
> > > Sure, but I see no conflicts, maybe someone already dealt with it?
> > > > @laliwa would it be possible for you to rebase this PR? It currently contains conflicts.
> > > > Thanks.
> > > 
> > > 
> > > Sure, but I see no conflicts, maybe someone already dealt with it?
> > 
> > 
> > By pulling the jakartaeechina repo and pulling your PR, I'm able to rebase against the origin/src. I can see that there is a few commits with conflicts, I tried to fixed them but there are some commits that contains chinese text changes and I don't know what is good and what is not good.
> > Let me know if you need help rebasing your PR.
> 
> I had tried to do this, pls check if things going well

There are still conflicts.

I can try to rebase it but you would have to review all your changes to make sure everything is still there.

laliwa commented:
> > > > > rebase
> > > > 
> > > > 
> > > > Sure, but I see no conflicts, maybe someone already dealt with it?
> > > > > @laliwa would it be possible for you to rebase this PR? It currently contains conflicts.
> > > > > Thanks.
> > > > 
> > > > 
> > > > Sure, but I see no conflicts, maybe someone already dealt with it?
> > > 
> > > 
> > > By pulling the jakartaeechina repo and pulling your PR, I'm able to rebase against the origin/src. I can see that there is a few commits with conflicts, I tried to fixed them but there are some commits that contains chinese text changes and I don't know what is good and what is not good.
> > > Let me know if you need help rebasing your PR.
> > 
> > 
> > I had tried to do this, pls check if things going well
> 
> There are still conflicts.
> 
> I can try to rebase it but you would have to review all your changes to make sure everything is still there.

Thanks alot for your kindness, but no worries, that is my duty to solve it till everything is OK :)
Now I rebased it again, I see the "Successfully rebased" which now shown the last time , I take that maybe a good hint.

ericpoirier commented:
@laliwa I see that the conflicts have been fixed now so I'm reviewing your PR one more time and I found the following issues:

In the documents section of the following page, we now have chinese translated text on the english page:
https://deploy-preview-1362--jakartaee.netlify.app/committees/

Remove the sidebar on the following page:
https://deploy-preview-1362--jakartaee.netlify.app/zh/connect/

Remove the buttons in the Jumbotron of the following page:
https://deploy-preview-1362--jakartaee.netlify.app/zh/membership/

Otherwise everything else looks good!

Thanks!

laliwa committed:
fix these found out by Eric Poirier ,thanks :

In the documents section of the following page, we now have chinese translated text on the english page:
https://deploy-preview-1362--jakartaee.netlify.app/committees/

Remove the sidebar on the following page:
https://deploy-preview-1362--jakartaee.netlify.app/zh/connect/

Remove the buttons in the Jumbotron of the following page:
https://deploy-preview-1362--jakartaee.netlify.app/zh/membership/

Signed-off-by: Eric <40311795@qq.com>

laliwa commented:
> @laliwa I see that the conflicts have been fixed now so I'm reviewing your PR one more time and I found the following issues:
> 
> In the documents section of the following page, we now have chinese translated text on the english page: https://deploy-preview-1362--jakartaee.netlify.app/committees/
> 
> Remove the sidebar on the following page: https://deploy-preview-1362--jakartaee.netlify.app/zh/connect/
> 
> Remove the buttons in the Jumbotron of the following page: https://deploy-preview-1362--jakartaee.netlify.app/zh/membership/
> 
> Otherwise everything else looks good!
> 
> Thanks!

Fix that, thanks a lot 👯
